tools/UpdatePaths/Scripts/74812_tree_repath.txt

```diff
@@ -0,0 +1 @@
+/mob/living/simple_animal/hostile/tree : /mob/living/basic/tree{@OLD;wander=@SKIP}
```

Comment:
```suggestion
/mob/living/simple_animal/hostile/tree : /mob/living/basic/tree{@OLD;wander=@SKIP}
/mob/living/simple_animal/hostile/tree : /mob/living/basic/festivus{@OLD;wander=@SKIP}
```

doing it like this allows us to account for potential downstreams who might have the festivus pole mapped in or something